src/main/java/com/futurcollars/lesson14/zad2/PeselValidator.java

```diff
@@ -0,0 +1,13 @@
+package com.futurcollars.lesson14.zad2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PeselValidator {
+
+    public boolean isValidatePesel(String numberPesel) {
```

Comment:
Dziwnie tą funkcję nazwałeś, może `isPeselValid`, albo po prostu `isValid`, bo zauważ że jak bedziesz tworzył zmienną `peselValidator` i wywołał byś na niej funkcję to miałbyś coś takiego: `peselValidator.isValid("123");` - krótko, czytelnie i wiadomo co się bedzie działo po wywołaniu.